v3.0/tidb-in-kubernetes/scale-in-kubernetes.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Kubernetes 上的 TiDB 集群扩缩容
 category: how-to
-aliases: ['/docs-cn/v3.0/how-to/scale/tidb-in-kubernetes/']
```

Comment:
This is for users who save the original link. We can keep it so those users can visit the document successfully.